### DISCUSSION TITLE
Arm on AUX1

### DISCUSSION THREAD

**Discussion by Vicking87** [ExpressLRS/ExpressLRS#2607] [Q&A]
Hi,
There are a lot off warning "Put your arm switch on AUX1" in doc.
I'm using Radiomaster BOXER how can i know witch switch is Aux1 ?
Thanks

**Comment by mha1** [comment]
It's channel 5, so use EdgeTX mixer channel 5 and set it up to use a 2-pos switch to drive ARM the way ELRS expects it.

**Reply by Vicking87** [reply]
Thanks,

So i have tu use channel 5 in EdgeTX Mixer, but i can use any switch i want ?

**Reply by mha1** [reply]
yes and yes. Use any physical or logical switch as Source

![image](https://github.com/ExpressLRS/ExpressLRS/assets/5615068/55f19c77-21ab-48af-bd1b-c937595ba72e)
